docs/Shop-Integration-Guide.md

```diff
@@ -0,0 +1,19 @@
+# Payone Integration Service Shop Integration Guide
+
+## Payment amount validation
+
+  1. **Never re-use payment objects in a cart** if cart line items are changed (added/removed/quantity).
+  Create a new payment object and add to the cart, don't update transactions/amount planned on existent payments.
+  Omit re-usage of payment objects once payment process has been initiated (transaction added
+  or customer redirected to payment provider page).
```

Comment:
```
**Never re-use payment objects in a cart** Create a new payment object and assign it to the cart if after payment creation any of the following occured:

- Cart has changed (items, address, quantity, total amount, ...)
- Payment has changed (type, amount planned, ...)
- Payment process has been initiated (transaction added or customer redirected to payment provider page)
```

Reply:
3548926 , but i moved the last point (`payment process has been initiated`) to the top description, because actually bullet points (cart/payment changes) are connected exactly to the case, when payment process has been initiated.